openmdao/visualization/n2_viewer/src/N2Arrow.js

```diff
@@ -968,4 +980,86 @@ class N2ArrowManager {
         this.pinnedArrows.removeAll();
         debugInfo(`removeAllPinned(): Removed ${removedArrowIds.length} arrows`)
     }
+
+    /* Save all pinnedArrows to a dictionary for saving the view. */
+    savePinnedArrows() {
+        let pinned = this.pinnedArrows;
+        let data = {};
+
+        for (var arrowId in pinned.arrows) {
+            let arrow = pinned.arrows[arrowId];
+
```

Comment:
Using `var` makes a global, so it's usually avoided. Using `let` is OK because it limits the scope to the block, but if you're not going to reassign the variable, `const` is generally the way to go. It creates a constant reference (with block scope), so you can change its properties, but not reassign it. Here, `const arrowId in pinned.arrows` and `const arrow = pinned.arrows[arrowId]` would be good.

Reply:
Thanks, I will fix these.